### PR TITLE
LPS-132788 Reorder the field display order

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
Hey @marcelabc ,

Could you fix semantic versioning occurred in relevant test?

Thanks!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
I forgot to add the feature flag in this PR

**Comment by rodrigopaulino** [comment]
@marcelabc, it was discussed later today that we'll have some changes in the Search Location's settings layout. Please confirm with the team what should be the new layout from now on.

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
Just started reviewing.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -1643,25 +1650,27 @@ private void _populateDDMDataProviderNavigationItem(
...
+		return stream.filter(
+			ddmFormFieldType -> {
+				if (Objects.equals(
+						ddmFormFieldType.getName(),
+						DDMFormFieldTypeConstants.SEARCH_LOCATION)) {
+
+					return _ffSearchLocationDDMFormFieldTypeEnabled;
+				}
+
+				return !ddmFormFieldTypesOutofScope.contains(ddmFormFieldType);
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:1669]: Hey, @marcelabc. I believe this was supposed to be `ddmFormFieldType.getName()`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 2]
```diff
@@ -1643,25 +1650,27 @@ private void _populateDDMDataProviderNavigationItem(
 	private List<DDMFormFieldType> _removeDDMFormFieldTypesOutOfScope(
 		List<DDMFormFieldType> ddmFormFieldTypes) {
 
-		List<DDMFormFieldType> availableDDMFormFieldTypes = new ArrayList<>();
+		List<String> ddmFormFieldTypesOutofScope = Arrays.asList(
```
Comment (rodrigopaulino) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:1653]: Could you please change `ddmFormFieldTypesOutofScope` to `ddmFormFieldTypesOutOfScope`?

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hey, @marcelabc.

I've made some suggestions.

Thanks for working on this.

**Comment by marcelabc** [comment]
Resent: https://github.com/liferay-data-engine/liferay-portal/pull/489

**Pull request closed without merging**